src/LiquidationPair.sol

```diff
@@ -439,10 +452,9 @@ contract LiquidationPair is ILiquidationPair {
   /// @notice Computes the current auction period
   /// @return the current period
   function _computePeriod() internal view returns (uint256) {
-    uint256 _timestamp = block.timestamp;
-    if (_timestamp < periodOffset) {
+    if (block.timestamp < periodOffset) {
       return 0;
     }
-    return (_timestamp - periodOffset) / periodLength;
+    return (block.timestamp - periodOffset) / periodLength;
```

Comment:
Storing the timestamp in memory saves about 12 gas in this function vs. accessing `block.timestamp` twice. It's more gas efficient to leave it as it was.

Reply:
I just compared them side-by-side and referencing block.timestamp directly saved 12 gas:

getPeriodStart (/w cache) median: 3127
getPeriodStart (direct access) median: 3115